packages/mermaid/src/docs.mts

```diff
@@ -41,12 +41,27 @@ import { remark } from 'remark';
 // @ts-ignore No typescript declaration file
 import flatmap from 'unist-util-flatmap';
 
+/**
+ * Windows uses '\\' as path delimitter.
+ * The package `globby` requires all paths to contain forward-slashes only.
+ * Also it is better if the `AUTOGENERATED_TEXT` has same paths (type of slashes) in all platforms.
+ * But while actually using the paths, they should be used with platform-specific delimiters.
+ * So replace all '\\' with '/' for `globby` and `AUTOGENERATED_TEXT` only to maintain uniformity.
+ * @param path
+ * @returns normalized path
+ */
+const normalizeUniform = (path: string) => {
+  return path.replace(/\\/g, '/');
+};
+
 // These paths are from the root of the mono-repo, not from the
 // mermaid sub-directory
-const SOURCE_DOCS_DIR = 'packages/mermaid/src/docs';
+const SOURCE_DOCS_DIR = join(...'packages/mermaid/src/docs'.split('/'));
```

Comment:
globby requires and returns paths with `/`.
Does mkdir and writefilesync work with `/` ?

In that case, instead of all the replacing, you could use `path.posix.join` which will join with `/`.
That way, we can remove this join completely.

Reply:
You are right. None of the replacing is needed, posix.join() solved all the problems. That's amazing!